Doppler.Currency/Services/CurrencyHandler.cs

```diff
@@ -39,8 +39,8 @@ public abstract class CurrencyHandler
             Exception e = null)
         {
             Logger.LogError(e ?? new Exception("Error getting HTML"),
-                $"Error getting HTML, title is not valid, please check HTML: {htmlPage}");
-            await SlackHooksService.SendNotification(HttpClient, $"Can't get currency from {currencyCode} currency code, please check Html in the log or if the date is holiday {dateTime.ToUniversalTime():yyyy-MM-dd}");
+                    $"Error getting HTML, title is not valid, please check HTML: {htmlPage}");
```

Comment:
Try use [structured logging](https://softwareengineering.stackexchange.com/questions/312197/benefits-of-structured-logging-vs-basic-logging) always.
Additional, again bad indentation.
```suggestion
                "Error getting HTML, title is not valid, please check HTML: {Page}", htmlPage);
```